### PR TITLE
Added config_scylla_manager_yaml(segments_per_repair=10)

### PR DESCRIPTION
**Pull request by yarongilor**
## PR pre-checks (self review)

- [x] I followed [KISS principle](https://en.wikipedia.org/wiki/KISS_principle) and [best practices](https://docs.google.com/document/d/1jihgOKb5iGRlD8_HQ92O0JbLk1kASUoZT23i_MXFSKI)
- [x] I gave variables/functions meaningful self-explanatory names
- [x] I didn't leave commented-out/debugging code
- [x] I didn't copy-paste code
- [ ] I added the relevant `backport` labels
- [ ] New configuration option are added and documented (in `sdcm/sct_config.py`)
- [ ] I have added tests to cover my changes (Infrastructure only - under `unit-test/` folder)
- [ ] All new and existing unit tests passed (CI)
- [ ] I have updated the Readme/doc folder accordingly (if needed)

### CODE REVIEW

**Comment by yarongilor** [comment]
This PR is planned to resolve https://github.com/scylladb/scylla-cluster-tests/issues/1447

**Comment by roydahan** [comment]
@ShlomiBalalis FYI

File: sdcm/cluster.py [hunk 1]
```diff
@@ -1863,6 +1864,7 @@ def install_mgmt(self, scylla_repo, scylla_mgmt_repo):
         """.format(mgmt_user=mgmt_user, rsa_id_dst=rsa_id_dst, rsa_id_dst_pub=rsa_id_dst_pub))  # generate ssh public key from private key.
         self.remoter.run('sudo bash -cxe "%s"' % ssh_config_script)
 
+        self.config_scylla_manager_yaml(segments_per_repair=10)
```
Comment (roydahan) [inline sdcm/cluster.py]: What is the default currently?
1?

Comment (ShlomiBalalis) [reply sdcm/cluster.py]: The value does not exist in the manager yaml by default, but from what I found in scylla docs, it seems like the default is 1

File: sdcm/cluster.py [hunk 2]
```diff
@@ -1894,6 +1896,15 @@ def stop_scylla_manager_log_capture(self, timeout=10):
         self._scylla_manager_journal_thread.join(timeout)
         self._scylla_manager_journal_thread = None
 
+    def config_scylla_manager_yaml(self, segments_per_repair=None):
```
Comment (bentsi) [inline sdcm/cluster.py]: The solution is error-prone. Need to bring the yaml, parse it, change the values dumps and upload.

**Comment by yarongilor** [comment]
@ShlomiBalalis , please re-write the parsing part according to Bentsi's comment.

**Comment by bentsi** [comment]
@yarongilor why did you close the issue?

**Comment by roydahan** [comment]
@ShlomiBalalis is working on different one.

**Comment by yarongilor** [comment]
@ShlomiBalalis , please update if there's a new PR to handle this. currently the issue is reproduced in ICS 1TB longevity.
```
(DisruptionEvent Severity.ERROR): type=end name=ManagementRepair node=Node longevity-tls-1tb-7d-2019-1-db-node-478a4bf8-2 [34.255.6.20 | 10.0.231.207] (seed: False) duration=86588 error=Unexpected result on waiting for task repair/0ad7aa51-4d18-41d0-80e1-35c1c3c942e8 status
Traceback (most recent call last):
File "/sct/sdcm/nemesis.py", line 1117, in wrapper
result = method(*args, **kwargs)
File "/sct/sdcm/nemesis.py", line 1332, in disrupt
self.call_random_disrupt_method()
File "/sct/sdcm/nemesis.py", line 624, in call_random_disrupt_method
disrupt_method()
File "/sct/sdcm/nemesis.py", line 902, in disrupt_mgmt_repair_cli
task_final_status = mgr_task.wait_and_get_final_status(timeout=86400) # timeout is 24 hours
File "/sct/sdcm/mgmt.py", line 266, in wait_and_get_final_status
raise ScyllaManagerError("Unexpected result on waiting for task {} status".format(self.id))
ScyllaManagerError: Unexpected result on waiting for task repair/0ad7aa51-4d18-41d0-80e1-35c1c3c942e8 status
```

**Comment by roydahan** [comment]
@ShlomiBalalis didn't you fixed it??

**Pull request closed without merging**